portfolio/src/main/java/com/google/sps/servlets/CommentHandlerServlet.java

```diff
@@ -13,16 +20,20 @@ public class CommentHandlerServlet extends HttpServlet {
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
         
         //gets comment text
-        String comment = request.getParameter("text-comment");
+        String comment = Jsoup.clean(request.getParameter("text-comment"), Safelist.none());
         String name = request.getParameter("commenter-name");
+        long timestamp = System.currentTimeMillis();
 
-        //log comment for later
-        System.out.println("Commenter: " + name);
-        System.out.println("Comment: " + comment);
-
-        //build response
-        response.setContentType("text/html;");
-        response.getWriter().println("Your comment was logged. Thank you, " + name + "!");
+        Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
```

Comment:
Can you move the datastore write to a separate method. 
writeToDatastore(comment, name)
reditect()